src/components/atoms/OcButton/OcButton.vue

```diff
@@ -330,7 +330,7 @@ export default {
   &-m {
     @include oc-button-line-height(1);
 
-    font-size: 1.125rem;
+    font-size: 1rem;
```

Comment:
wouldn't it make sense to adapt the font sizes of the other size-classes as well?